{{ cookiecutter.repo_name }}/build.gradle.kts

```diff
@@ -20,7 +20,8 @@ plugins {
 allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        gradlePluginPortal()
```

Comment:
Good to know which plugins requires to use `gradlePluginPortal()`. It would be good if you can share more info on this change. Thanks.

Reply:
we are using gradlePluginPortal() to do not redirect to JCenter to check repositories and also  not getting error from any  transitive dependency plugin after replacing to mavenCenteral().

Reply:
`gradlePluginPortal()` does redirect to `jCenter` as a fallback if plugin not found in gradle plugin portal. See [detailed info](https://github.com/gradle/gradle/issues/4020#issuecomment-357489131) on it. If our purpose is to completely get rid of `jCenter` then you have to identify which plugins still require `jCenter` as fallback.

Reply:
@prem-p-simform i have go through and find out , we are using this library br.com.simplepass:loading-button-android:2.2.0 loading button he is trying to finding this one and getting error 
Searched in the following locations:
  - https://dl.google.com/dl/android/maven2/br/com/simplepass/loading-button-android/2.2.0/loading-button-android-2.2.0.pom
  - https://repo.maven.apache.org/maven2/br/com/simplepass/loading-button-android/2.2.0/loading-button-android-2.2.0.pom
  - https://jitpack.io/br/com/simplepass/loading-button-android/2.2.0/loading-button-android-2.2.0.pom
  
  in this pom getting 404 error so if remove this lib we do not require gradlePluginPortal() else for this one we require gradlePluginPortal().

Reply:
Removed gradlePluginPortal() ,we removed this br.com.simplepass:loading-button-android:2.2.0  that depend's on it